analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -135,17 +135,22 @@ class Analyzer(private val config: AnalyzerConfiguration) {
                     val results = manager.resolveDependencies(files)
 
                     // By convention, project ids must be of the type of the respective package manager.
-                    results.projectResults.onEach { (_, result) ->
+                    results.projectResults.forEach { (_, result) ->
                         val invalidProjects = result.filter { it.project.id.type != manager.managerName }
                         require(invalidProjects.isEmpty()) {
                             val projectString = invalidProjects.joinToString { "'${it.project.id.toCoordinates()}'" }
                             "Projects $projectString must be of type '${manager.managerName}'."
                         }
                     }
+
+                    manager to results
                 }
             }.forEach { resolutionResult ->
-                resolutionResult.await().forEach { (_, analyzerResults) ->
-                    analyzerResults.forEach { analyzerResultBuilder.addResult(it) }
+                val (manager, results) = resolutionResult.await()
```

Comment:
`resolutionResult` should be renamed because `ResolutionResult` is the type of the variable now called `results` which is confusing.

Reply:
I renamed `resolutionResult` to `deferredResult` and `results` to `managerResult` (as it is of type `PackageManagerResult`).